src/main/java/internetshop/Main.java

```diff
@@ -0,0 +1,42 @@
+package internetshop;
+
+import internetshop.lib.Injector;
+import internetshop.model.Item;
+import internetshop.model.User;
+import internetshop.service.ItemService;
+import internetshop.service.UserService;
+import internetshop.service.impl.ItemServiceImpl;
+import internetshop.service.impl.UserServiceImpl;
+
+import java.util.List;
+
+public class Main {
+    static {
+        try {
+            Injector.injectDependency();
+        } catch (IllegalAccessException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public static void main(String[] args) {
+        User user1 = new User("Tom", "1234");
+        UserService userService = new UserServiceImpl();
```

Comment:
```suggestion
        @Inject
    private static UserService userService;
```
створи так усі 4 сервіси в класі, як поля, щоб перевірити, чи працює інжектор